src/vcy/interp.ml

```diff
@@ -624,6 +624,26 @@ and interp_stmt (env : env) (stmt : stmt node) : env * value option =
     end
   | Assume _ | Havoc _ ->
     env, None (* We simply ignore 'assume's and 'havoc's at runtime *)
+  (* | GCommute (variant, phi, pre, blocks, post) ->
```

Comment:
Does GCommute just error on runtime? (Should be fixed if we merge with the Commute case as noted before).

Semantics for pre/post conditions? Do we want those to behave like assumptions/assertions and error if they aren't satisfied?